src/equations/3d/hyperbolic_diffusion.jl

```diff
@@ -31,6 +31,7 @@ get_name(::HyperbolicDiffusionEquations3D) = "HyperbolicDiffusionEquations3D"
 varnames_cons(::HyperbolicDiffusionEquations3D) = @SVector ["phi", "q1", "q2", "q3"]
 varnames_prim(::HyperbolicDiffusionEquations3D) = @SVector ["phi", "q1", "q2", "q3"]
 default_analysis_quantities(::HyperbolicDiffusionEquations3D) = (:l2_error, :linf_error, :residual)
+default_analysis_errors(::HyperbolicDiffusionEquations3D)     = (:l2_error, :linf_error, :residual)
```

Comment:
It's not really a perfect solution, I agree. I use this right now because it is handled differently from other stuff in `default_analysis_integrals`. Currently, the residual is only used for hyperbolic diffusion, where it is indeed an error for the underlying Poisson problem. Do you have a better suggestion how to handle stuff like that?

Reply:
No, I don't have any concrete proposal right now. Maybe something more general, maybe along the lines of a termination criterion for iterative methods? Should we farm this out to an issue?

Reply:
Yeah, I think it's good to track this in an issue. It's working right now and not really blocking, I think. Will you do that, please?

Reply:
#242